apps/snp_store/snp_store.cpp

```diff
@@ -703,82 +734,161 @@ int detectSNPs(SNPCallingOptions<TSpec> &options)
                 return 1;
             }
         }
+    }
     // Allocate one BamAlignmentRecord for each input BAM file.
     std::vector<BamAlignmentRecord> records(length(options.readFNames));
 
     /////////////////////////////////////////////////////////////////////
+    std::time_t t = std::time(nullptr);                                         //Get time and date for vcf header
+    char now[24];
+    strftime(now, sizeof(now), "%Y%m%d", std::localtime(&t));
```

Comment:
I cannot compile or run the code where I am working from at the moment, hence I would almost certainly introduce some errors if i tried to switch to chrono library now.
As for the time stamp: Yes it is common to write it to the header of the vcf file (see https://samtools.github.io/hts-specs/VCFv4.2.pdf). I cannot change this now for the same reason as mentioned above.